Jamulus.pro

```diff
@@ -68,6 +68,35 @@ INCLUDEPATH_OPUS = libs/opus/include \
     libs/opus/silk/fixed \
     libs/opus
 
+# Add include paths and headers for new audio APIs here
+
+# ASIO
```

Comment:
I'd split this into two parts:
- provided by the SDK
- "part of Jamulus"

Reply:
I'd also make _all_ the variable definitions be either
```
VARNAME =
```
or
```
VARNAME = value
```
depending on the config that makes them relevant.